client/src/components/OutlineButton.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Text, Button } from "@chakra-ui/core";
+import { Link, useLocation } from "react-router-dom";
+
+const OutlineButton = ({ text, to, width, height, onClick }) => {
+    const location = useLocation();
+
+    return (
```

Comment:
CC @FocalChord - I moved your `HomePageButton` to a shared component since it's being used in multiple areas of the application

Reply:
Yup, thanks for that!